src/components/Pair/Hero/ExchangeWidget/ExchangeWidget.js

```diff
@@ -8,7 +8,7 @@ import axios from 'axios';
 import config from 'Config';
 
 import { setWallet, errorAlert, setOrder, setDestinationTag, setPaymentId, setMemo } from 'Actions/index.js';
-import { bindCrispEmail } from 'Utils/crispEmailBinding';
+// import { bindCrispEmail } from 'Utils/crispEmailBinding';
```

Comment:
I see this commented out in a few places.
this we can leave as a comment as we might come back to using crisp.

Reply:
While uncommented, this line gives 'unused var' warning, which breaks travis checks.